packages/vike-node/src/plugin/plugins/serverEntryPlugin.ts

```diff
@@ -49,9 +49,6 @@ export function serverEntryPlugin(): Plugin {
           config
         )
       }
-
-      config.vitePluginServerEntry ??= {}
-      config.vitePluginServerEntry.inject = Object.keys(resolvedEntries)
```

Comment:
Upon `inject` the Vike and Telefunc entry are directly injected inside the user's server entry. One neat thing about it is that it skips the whole [auto importer logic](https://github.com/brillout/vite-plugin-server-entry#how-it-works) which is brittle. So, unless there is a reason, I think we should keep `inject`.

Reply:
It was breaking the whole thing with this code at some point, but at do not manage to reproduce the issue. So I'll add it back :+1: